extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -18,27 +18,27 @@ import {
   CONSENT_POLICY_STATE,
   CONSENT_STRING_TYPE,
 } from '#core/constants/consent-state';
-import {Deferred} from '#core/data-structures/promise';
-import {PauseHelper} from '#core/dom/video/pause-helper';
-import {Services} from '#service';
-import {addParamsToUrl} from '../../../src/url';
-import {applyFillContent, isLayoutSizeDefined} from '#core/dom/layout';
-import {dict} from '#core/types/object';
+import { Deferred } from '#core/data-structures/promise';
```

Comment:
Your auto-formatting configuration may be wrong, since it's inserting these spaces in destructure statements. Please fix so that this can be merged.

Reply:
updated